ezbake/config/logback.xml

```diff
@@ -18,6 +18,7 @@
 
     <root level="info">
         <!--<appender-ref ref="STDOUT"/>-->
+        <appender-ref ref="${logappender:-DUMMY}"/>
```

Comment:
Since this works without the `-DUMMY`, and logback doesn't complain about it, I think we should remove it. I think the code will be cleaner, more readable, and less confusing.

I also think it would be worth adding a comment explaining that this is used to log to the console when running the `foreground` command.